addon/popup.js

```diff
@@ -83,7 +85,7 @@ function renderTabList(tabs, containerSelector, eventLabel) {
       image.style.backgroundImage = `url(${favIconUrl})`;
     }
     renderedInfo += favIconUrl + " ";
-    let anchor = document.createElement("button");
```

Comment:
Same here, if this is a button then it can be navigated to using the keyboard. Switching it to anything else will regress the bug I fixed by switching this to a button.